client/src/main/proto/spine/client/entities.proto

```diff
@@ -73,8 +74,26 @@ message EntityFilters {
     // Match the entities by the value of the Entity Columns.
     map<string, google.protobuf.Any> column_filter = 2;
 
+    // Match the entities which are created later then the given time.
```

Comment:
I am not sure, but is it just the creation date or the date of the latest modification as well? I understand that for `EventRecord` it's just a creation. However, as long as this is a generic API, we should mention the "last modified" timestamp as well, i.e. how it is affected by this filter.

Also, depending on the details you add to this description, I would rename the field itself. If it is still just a `creationDate`, let's call the field `created_after`. If the filter also affects the modification date, it should be `modified_after` (assuming that for `EventRecord` the creation date and the date of the latest modification is the same).

Let's discuss _vocally_ if needed. Otherwise, just add the requested changes and let me know.